backend.native/tests/produce_dynamic/simple/hello.kt

```diff
@@ -96,3 +97,15 @@ fun useInlineClasses(ic1: IC1, ic2: IC2, ic3: IC3) {
     assert(ic2.value == "bar")
     assert(ic3.value is Base)
 }
+
+fun setCErrorHandler(callback: CPointer<CFunction<(String) -> Unit>>?) {
```

Comment:
`CPointer<CFunction<(String) -> Unit>>` is not supposed to work this way.
Use `CPointer<ByteVar>` instead of `String`.